### DISCUSSION TITLE
Sharing header styles (fonts and colours) across all Navigators

### DISCUSSION THREAD

**Discussion by pke** [react-navigation/react-navigation#10974] [Q&A]
Is there a built in way to share header styles not only among the screens of a navigator but across all navigators inside the `NavigationContainer`?

**Comment by traviswimer** [comment]
Did you find a solution to this? I couldn't find any directly supported way to do this, but there are a few ways to accomplish it.

The simplest solution is probably to use [themes](https://reactnavigation.org/docs/themes/), but this only works for color and it affects more than just headers.

Otherwise, you could use a custom navigator or create a simple wrapper component that creates a navigator with the default styles you want.

**Reply by pke** [reply]
No, this is still not possible without custom navigators.
